packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/internal/span/TextInlineImageSpan.java

```diff
@@ -5,7 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-package com.facebook.react.views.text;
+package com.facebook.react.views.text.internal.span;
```

Comment:
This is problematic as `ReactTextInlineImageShadowNode` (which is public) has this method:

```
public abstract class ReactTextInlineImageShadowNode extends LayoutShadowNode {
  /**
   * Build a {@link TextInlineImageSpan} from this node. This will be added to the TextView in place
   * of this node.
   */
  public abstract TextInlineImageSpan buildInlineImageSpan();
}
```

so we'll be returning an `.internal` object from a public class.

We probably should move also `ReactTextInlineImageShadowNode` to `.internal`

Reply:
Done

Reply:
@cubuspl42 can we make sure the CI is green?